packages/aserto/src/aserto/client/directory/__init__.py

```diff
@@ -1,14 +1,29 @@
-from typing import List, Literal, Tuple
+from typing import List, Literal, Tuple, Any, Optional
 
-from grpc import ChannelCredentials, ssl_channel_credentials
+from grpc import ChannelCredentials, ssl_channel_credentials, secure_channel, Channel
+import grpc.aio as grpc_aio
```

Comment:
The sync directory client should not be importing `grpc.aio`. Only the async client should import that.
That's the reason we split the async client into its own module.